zkevm-circuits/src/root_circuit.rs

```diff
@@ -142,4 +142,4 @@ impl<'a, M: MultiMillerLoop> Circuit<M::Scalar> for RootCircuit<'a, M> {
         Ok(())
     }
 }
-                max_inner_blocks: 64,
+                
```

Comment:
We can remove this to limit the diff. As the `root_circuit.rs` module is not included in the module tree, I don't think it should be an issue, right?